pkg/KubeArmorOperator/PROJECT

```diff
@@ -0,0 +1,19 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+domain: kubearmor.com
+layout:
+- go.kubebuilder.io/v3
```

Comment:
Should we move to go/v4. :thinking: 
Ref - https://book.kubebuilder.io/plugins/go-v3-plugin.html

Reply:
done :+1: